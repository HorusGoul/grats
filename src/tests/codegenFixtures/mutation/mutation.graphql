type Mutation {
  setGreeting(greeting: String): String
}
