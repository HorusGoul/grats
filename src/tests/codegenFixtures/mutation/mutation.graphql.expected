-----------------
INPUT
----------------- 
type Mutation {
  setGreeting(greeting: String): String
}

-----------------
OUTPUT
-----------------
import { GraphQLSchema, GraphQLObjectType, GraphQLString } from "graphql";
const MutationType: GraphQLObjectType = new GraphQLObjectType({
    name: "Mutation",
    fields() {
        return {
            setGreeting: {
                name: "setGreeting",
                type: GraphQLString,
                args: {
                    greeting: {
                        name: "greeting",
                        type: GraphQLString
                    }
                }
            }
        };
    }
});
const schema = new GraphQLSchema({
    mutation: MutationType
});
export { schema };
